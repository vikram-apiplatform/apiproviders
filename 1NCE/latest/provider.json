{
    "producer": "1NCE",
    "title": "1NCE",
    "description": "IoT Connectivity & Software for Life - 1NCE is the global Tier-1 IoT Carrier specialized in providing managed connectivity services for low bandwidth IoT.",
    "image": "https://s3.amazonaws.com/services.apiplatform.io/service-providers/Service%20providers%20-%20resized%20logo's/Resized%20Logo's/1NCE.png",
    "url": "https://1nce.com/en-eu/",
    "authorization": [
        {
            "authtype": "Bearer Token",
            "authattributes": [
                {
                    "key": "token",
                    "displayname": "Token"
                }
            ]
        }
    ],
    "partner": "services",
    "account": "services",
    "isPromoted": false,
    "id": 336
}