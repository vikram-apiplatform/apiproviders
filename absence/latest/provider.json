{
    "producer": "absence",
    "title": "absence",
    "description": "Easily manage absences, employee information, and time tracking with just one tool.",
    "image": "https://www.absence.io/fileadmin/images/absence-logo.svg",
    "url": "https://www.absence.io/",
    "authorization": [
        {
            "authtype": "API Key",
            "authattributes": [
                {
                    "key": "key",
                    "value": "",
                    "description": "Key",
                    "displayname": "Key"
                },
                {
                    "key": "value",
                    "value": "",
                    "description": "Value",
                    "displayname": "Value"
                }
            ]
        }
    ],
    "partner": "services",
    "account": "services",
    "isPromoted": false,
    "id": 511
}