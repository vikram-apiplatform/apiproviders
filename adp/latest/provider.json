{
    "producer": "adp",
    "title": "ADP Workforce",
    "description": "ADP workforce now is a cloud-based platform for handling payroll, time and attendance, talent management, benefits administration, and other HR processes. It enables organizations to efficiently manage the complexities of the employee lifecycle. Built on a single, configurable database, it facilitates data syncing across different departments to reduce manual entry and data errors. Its embedded with protective layers such as physical security, intelligent detection, fraud defence and other mechanisms for enhanced security and data privacy.",
    "image": "https://s3.amazonaws.com/services.apiplatform.io/service-providers/ADP.svg",
    "url": "https://developers.adp.com/",
    "authorization": [
        {
            "authtype": "OAuth 2.0",
            "authattributes": [
                {
                    "key": "base_url",
                    "displayname": "Authorize URL"
                },
                {
                    "key": "token_url",
                    "displayname": "Access Token URL"
                },
                {
                    "key": "refreshtoken_url",
                    "displayname": "Refresh Token URL"
                },
                {
                    "key": "client_id",
                    "displayname": "Client ID"
                },
                {
                    "key": "client_secret",
                    "displayname": "Client Secret"
                },
                {
                    "key": "scope",
                    "displayname": "Scope"
                }
            ]
        }
    ],
    "partner": "services",
    "account": "services",
    "isPromoted": false,
    "id": 199
}