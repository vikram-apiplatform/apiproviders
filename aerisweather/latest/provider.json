{
    "account": "services",
    "authorization": [],
    "description": "An advanced weather API to power all of your business needs, from the basic to the most complex weather influenced solutions.",
    "image": "https://s3.amazonaws.com/services.apiplatform.io/service-providers/AerisWeather.png",
    "isPromoted": true,
    "partner": "services",
    "producer": "aerisweather",
    "title": "AerisWeather",
    "url": "https://www.aerisweather.com/support/docs/api/reference/endpoints/#all:all",
    "id": 85
}