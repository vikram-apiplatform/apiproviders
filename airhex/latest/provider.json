{
    "producer": "airhex",
    "title": "AirHex",
    "description": "AirHex help travel companies to get rid of routine by providing airline-related data and updates right to their web platform",
    "image": "https://res.cloudinary.com/crunchbase-production/image/upload/c_lpad,f_auto,q_auto:eco,dpr_1/v1475731262/yvxov6n5a9crb56ysvsq.png",
    "url": "https://airhex.com/",
    "authorization": [
        {
            "authtype": "API Key",
            "authattributes": [
                {
                    "key": "key",
                    "displayname": "Key"
                },
                {
                    "key": "value",
                    "displayname": "Value"
                }
            ]
        }
    ],
    "partner": "services",
    "account": "services",
    "isPromoted": false,
    "id": 455
}