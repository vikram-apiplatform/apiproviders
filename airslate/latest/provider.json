{
    "producer": "airslate",
    "title": "airslate",
    "description": "Empowering anyone to create, innovate, automate, and digitally transform their workflows",
    "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAVAAAACWCAMAAAC/8CD2AAAAkFBMVEX///8sQ1zg4ODc3NwoQFoXNVIiPFcaN1MmPli4vsV3g5K8wslba378/P2or7gfOlX09vdNXnLn6ewSMlAAK0tzfowLL0729/hBVGqnrreutb3P09hseYnHzNEAKEnZ3OCDjps1S2Pr7e9SYnZhcII/U2qZoq2QmaV+ipiVn6oAI0ZndITLz9SKk58ySGAAHkIu+ugvAAAK2klEQVR4nO2be4OqKhfGczZ4KRsty+w2Vpbaaep8/293NAHRpAuz2u9+z1nPXzOGCD+BdQF7HyhQ9X71UID6hUBhhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCBhUCB9ecCHf6vG6CndwOdDc4Hd/bSLfHifEqiw3Z7GLnTbPHtv6lp79E7gc6Wx2QSeCQcPH/PMrdCyzNJJdv0rGCy3Ti7tzUSWm8EujZCjxiF6NNA10loGjciZmCu/l9WgDcCPXE2zwId9kP7FmelyfpdzQTWG4Hu7deA7kaWCqdhWHzS7z502hL/tjXjdwB9bg31tx2zncs+sVJjm0b54FU71Z8Y+3n84k16ejtQ2wqTp/of3eFZD3InKJZU+jU6Ll9pSmHkinaYufOav6GjdwI1iReEbvrcyJjTOzwNm78TpypGvL8WLzSFVDeZoftyL17VO628vd0Mnh0SvkFkgKXHZNuEXzOnvJzDuZv9F5oi6rbe7tT+KZHSpzxArWA7Oq1ObnQJA6tcOAJh438I1PvPAHVrh8n05uPZ1e0c+rvvuTsJLFOU+3cDncVOmuXTvbvqn9PB+EXfe7b8zPL+vLIuk5rnqlVuONjUy7AC6NBfLtKsv3LdVZ4uvttOkgBqqtrof8+zftGRzfzlfjSlDzROT0ZIizCxWOzMIkSkYZSNb4vt1pu0+muYTUv1q97usm0ZY5rW1brEoVg9L3eX3S6g/nozCgKrqM0uZHo0CNymPRdAyWpayZF/nn3uraDqSdmPw1nL162kC3R+oZbZsCOl6aX9xtsdfh9HduB9fV//cyZmqSAv/xmYFrvbKnl/B8Ken24fJqkDaE6o12qJYQcX2QuofzaZQmnupwZtRGhFP6bagYAm0B1td4HR2daDdLwPg2tPmRP5WQVCdum6pF/iflrO57UASkZ3H3wLdPnV2RQSSkmu2xKWaOf40NEXz3ohodOQJtBh1A3UsC+ipVMeSTaBkoPfSwU/g2yvVOoL3t2o/RboWBUQeLXTedtYypsZk878AQle8XMl6U75VBV322KEua1YngEtwvJ1bYKqGS+tocWV+R2r0DHl+yqiNONF1EDHgWJkGBPn9ulPSBfoLlS0wwh4Q1RA6WArdSKsLLiE2KCXbKmyTB1ApdHdUsiHoRLo7KDMbxGvw8Q+lraVX5XjwvboVZbULBKxEiqghhwU8dKu3LHCKlxO3UnlLitfvp8iVq+a4kkVeTkrUT+QlQov1Q8bS37o9f7axbpvHhXSBrqcFF1wj4P1crkepCvDq18+8xrVQFkPCjeFUOZTOUH7Rzq5ZLfbH11A56FNjVU6WBZNcY77Gimx2eIhINnOenBVVbG00liHbB3HceGXiJht8lIChknfD81GqTQndhvRDmteXboLlNAwWa1cErFx2GXliBdepi1r2wV0uHIdifx4L14uZUjqUddcSqZi+Q2OYuEW7bwJMZ4RXOiZe62u3gPqRe20yXfnSlgMVCOT5/5ToWcinvxZXVCEnv6XIJ9Klwe8obZGtg8O6JgDtZPqwh2g5v72/rnCzBGLzutSTwFdi0JsEVUAFZW1nF8+cOn36xgAkyPcdpND9WLvGKVLV45iPlF5MLROUT8FdMyfbLNJqwC64jPemjfu57PFSnsvCxDoiAM1qlargVrdTvNyq3RuTb5cPwV0xv0Im80EBVCxbtOWh8QmS52GfV4/AjqMF9mmP+1vUme56yUiHfwIqKlIos1SUxHS2lt2iwqoH8+zfFo2ZRD7vQt/t2z16Qa6E5cv8bghNtlsjQT/D4CuV0bhthW+T5lqovZoKwbgA6C8mx2aLUa3mY4rCTZ5O4EOHZdUea9rU4yE30VGle3uBvpRJ7WJ3VDr9lek74dGod3ctRB/PQLqne9VHGejoAPqpPJuu4A6RtCMd+ps3V2gsTLa47dvXzfzukA/1WcSHgK1Pu/XPRw758OENrODDGAH0E2oMmYPgKpj1t8PVOXjPAf0mUTOzDld5AWVde4WaH4Hy4MR+hBo9Lum/FLOZZBiAWqMkUdA6ZOZMX9xqANag17T6DdAG6+WSDulTwB9NOVN9WKvlB7QOk70qHVI9snFsurhBAW0DCprotXOZxvoTgSPRQDgHdx9UuY/1EDlOSwZJc/qUvhgbeqSFlCRXyfeme+fjdcH3mw4oPKrqxaKNlCRlyVkzpzJYeyogXa7TWQ88zuks1unBTQXEYacXU/eAfQshmg3UBG227JvLoK2u0BnYggEcOeetIBydOam6yooUPHuWC0toD7n3TR0l6eA1i30sh6UdIDO+ABonlP8IdD4NL8NoIa1T1mNohbQMbcrYSN4VI7QZvruyF8HIWAHIHSA+t1ARz8DevDo5WZDvDZK5HLN4j0FdKgcoVYD6FKYea3UZ6e0Rihfe7yjdLXefdQCGtPSxk2iNBZ99gdRbeRZxq8FdDdpPqPSWlgqBlSM8+ZA7tU+lnUCGqN6ayhDxfNKpXxhHvSAsulHrHDr9jfZcdPfE3lHMqgqaa+h/C3KGc2x2ATkQIWv0FosU8lxunxKsP3lppnRe1paQMXGrR2xPYahY9ShqA7Q4aj2YWzTK2Q2T3Nsq3ItoPVtpst4+PPaI+ZA662O5rNn0v4rKdzY0zn9XCwWx5FNvYnexrwW0EXtOYejPDse3UCOEnWAPopaQuagtd2mjVgU7NDNszRLQimpyoHWDTao0T+f84RX14g+iVk4+Hzn88ERFpW0gO6sRiM8r5ko0QF69Ix7CjasXBuovBV1HdnNDBgDKp8iKBtsmoTVt1J/KBHobHpqhp4bdTM0gTYPMLdFhRG+ieUTddZLSmj22++LiuMYyq68dAJVSA+o39X/n1j53d/3sIRS7rMNtHOtEP4lBzpunxTkO4m9Ya5cbKiO4ddM38VWe2CQoL+k2kB7eWApxiixiOQS3abvFjd7e3aYLqwm0N66Xaqe0IvuPQLD+NLY9NROMMeHxlcbJDgsej7zp2m1kd4GOr+bD92lUXDzlgpuwSWVcxTCitQTckAa89kOk2XvI2wB7Tlec9ZL2wb+mdD2gTPiBZ7W55DaWyCzoxV49vUTV9MKo0X58P7kem4oqRqSf1XniIK/q9TD8iuo/v+rOxUxjFM3DK+7VOzLWRrSaevTonHIK6nfyi4PaXVPccvELQ34cBSWhb5y6c5+uQlQ1exZwZccCewWp6B4Mn+uFUySbKB35vYnu56Dszs6bA9JX3ykVh0cYgiG7BjRgP8as//vnGobxoNjfkpGURSN9ptFh50ds0oaP82cjRttt1GSL1jls2uh5lFTf5CXFUfJ6vy5vFkfY+e4Sq6VTNO1ftj04335P+grYYim/LiOP+Wzmn+NECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwECiwfvU+UKD6B9mKrszJCwwhAAAAAElFTkSuQmCC",
    "url": "https://www.airslate.com/",
    "authorization": [
        {
            "authtype": "OAuth 2.0",
            "authattributes": [
                {
                    "key": "base_url",
                    "displayname": "Authorize URL"
                },
                {
                    "key": "token_url",
                    "displayname": "Access Token URL"
                },
                {
                    "key": "refreshtoken_url",
                    "displayname": "Refresh Token URL"
                },
                {
                    "key": "client_id",
                    "displayname": "Client ID"
                },
                {
                    "key": "client_secret",
                    "displayname": "Client Secret"
                },
                {
                    "key": "scope",
                    "displayname": "Scope"
                }
            ]
        },
        {
            "authtype": "Bearer Token",
            "authattributes": [
                {
                    "key": "token",
                    "displayname": "Token"
                }
            ]
        }
    ],
    "partner": "services",
    "account": "services",
    "isPromoted": false,
    "id": 660
}