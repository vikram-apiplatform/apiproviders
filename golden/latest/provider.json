{
    "producer": "golden",
    "title": "Golden",
    "description": "Golden is on a mission to map human knowledge. Our vision is to create a knowledge graph protocol that maps 10 billion entities and the public knowledge surrounding them.",
    "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAdoAAABqCAMAAAAsh2BcAAAAsVBMVEX////wtTRTU1NKSkpJSUlPT09CQkK0tLRGRkaWlpagoKD7+/tjY2OcnJykpKR4eHiqqqrOzs7s7Ox/f3/vsiaIiIjvsBvvsiP32aLvrxD77dPW1tb+/PbDw8NdXV1paWne3t7zxWr99+twcHDy8vLxuUDxu0f658b88uCOjo710Iv0yXfk5OT547z1zYPywV3426i7u7vyvlL0x2/43rD21Jb0ynr879n55L43NzcxMTG7KtAUAAAO1ElEQVR4nO1de0P6Og8e7MIQFNG5MQQEFES8oAjqeb//B3vXdhu7JG13Ezxnzz/n/Jgp2542SdMkKEoGrF4Hb7PtotFoLJab57f963CeRbzGSWL0uF66tmNZjRCW5diuPRsMj31vNfJjtNu4ToTUGCzbfv459h3WyIXhs+sgtIbsOuvVsW+zRla8LmxsvcbYdWffx77VGlmwa9gSvPrkftZO1Z/B41KaWEbu4Nh3XEMK85mbhVgCZ1sv3D+AHeoTc8l9PPZ91xBgtMmkiw9w34996zW4GDbyLFnG7f7YN1+Dg/fMVjYCu3amThdvOZVxsG5fj/0ANRDMBMEn8bqtQ1OniU1uMxti4Q1TnwqdHLbFmW1Yb95Am9GxH6VGDCWsWQ/ut6I8WjW3p4TPUphtNLbeWMvFsZ+mxgHrYr7xAfaPory6m2M/T40Ar0X2s3EsveEs5+vYT1SD4bs8Zhv20NMBlr079jPVoCiPWM9Jfvamit1w6y3uKeCrJBeKwR6xuXLsp6rhbVZKVMcenB+ikdkWt8ZxUSqxnkb2XKhHz+F262TWY+NNQh1bIWTIVZQR2Uvl391ejdu9SVNvTnrt8VVe0fPxg/jPx+3z8/N2P9NXdIhMAmcXrf6LjHAfEI6jDY1zSa9wH4iO3B5HPlnx1bHl2HZj+7ne7weD/f5ttm24toBgd64oS++/Ts7D+Y97w1T1JoGuaka31cknaho9IWdnhqqqxmWm++v8owLQTEO/HguFLw1IOArjFhDrmd4V80Y4snEe+YQTYLQcd/G2+06EDefD9+cGL5GV5NJQVeDkCTh+NE3GTQBdUyVffVrUnAjIvVC9P5Md30fHaCLQTfVCMA8vVUw4gAlSSx/M4D0NHVk9O3wwRBetY8926CHO6n3romUFA5JgRf4ne97F1Z0JPe1EQtnBokaP+67LpZaMpX5whQtR29Q5D5OidovQ4253gjU3HyC5ysSPGtJLdtZl2w+Xna6ZZqBbvX8ZQlWXENXCf6lTjlQBavU4DpPpmifMqNU5gBUyG199EowcoXYIsmO5X1IRh8cNuOZnHu92sH6z4MMImJw8tcb9cesmVLFGK6Po08TwFwhPjeWntnsdRe+u6Vl5f+HeidaW3rvG0ZsCYr3gNUC8R0aOUDsDtKrlvkmfpg+hBMil5yKzhA1LdhyKsb8cjKdp+Nnttc8Zf91Coi9tQxe9j9zUqufJjx/6bVVjd3GHC1MCDAnfPY5eqIT4Ix+onQOrzp5lihE+LlJpN2TXw6aMnSU7ecro0e7ihvW26RPE2QYholf3mkA0P7Vt6NIluw8Vd2UZtRl3dBFq4e8NRz5Qu08tWsvKXFg5SM0PJYyDzOSH6firM3XvnTtmn7q4rI6IKk8mX7RkapUXNg9xFVOUWlwFJahNMWt/Zv1SD6ullaJ2wf7PlXekzimB5hlw6Y4+mIn6nhzRG3YJs9RlU6s8+BxgwoWpRYeOU/uYNJRuzrO4RJarElLrSA/4QHWZDrqXHZ1rZ3ii/rRQEdHSqfVtAzoNC1HLHiVl5SMjh9SmYozEyu7yxH53UaVMTuMXWTUy2xVosHPZ574vJmrCoi9MFNGQ5VOrnKk8T6oItTpTQcYUHzmkNs6stSDac7bO+rUU0V0USZ8JNbzsABM6Iy+Qq+zRkPfFF72hoj34YgXUdqh5N5AgSxFqzdtrum4n+MgBtfFNrUUy1uaLvEH9VegoW8ReB7PGltQBVwbXlZ1yLgtEb9mrhi9WQK3SJmNqiHUvRG2fTRsNciri1A6sJLMrK/9RXMgtiS+OglkjG7UYm/h8JOiSZ9NAjSwSpZfNKXitCmrpXEIsf0Fq/f07qBJi1M6ii5Zo45VjZditJBEoAeI6rUJqJQek71jH42iXZIcK7xdFovd0VsDGtgpq2VxCHNmC1PqWCdrMxaiN6mNr7mlji6aI58arHergx1A9O3Ky1CKqeDRxqk88gMaWimIKUPEVJGKKK6GWOuVIxKkotWw3YAJ3HKV2FaHWffSUqO8B5QfzuMle9qDrJY8I7oN7zw4mischWxq+qiuhls4lxAIUpVb5wLy0KLU/hwihQ47fNlbDKVhGSbc8ZO9ziE3bcnqAerngeZYQTYEotcWIi1wJtXQuIdO0MLVsJgObhSi1ES+KsDGwcx6eR0DMLc13OwwtOVuavB0DH9zNhkKO+/CNUyXU0rmEGPfi1PoqOWWAotQeclRJXSxJpCniRDE8WzSj8fug6y25NBrGT+YTEQL2qPjbuuV40JVQS78QMf7FqVVaJrjZi1K7Cd8+iVJs5WngwLPfxLhGjh0suRAI46cAtbjoC8djrYTaqYkPWgK1zE3T76GRfWoPOnPke7ey8QUOZhbxxBaNA7Vy5T/5qWUn40iYkYCFGuEAdCXU0i9EXPIyqL0CVXKU2nDrSXyoRuDbFsQrOWGIhrksqbMkxg8SQZYR5VDL3gR8QFAJtVcVU6tcmsBCiFLrRhYtTVPLmBQBYk6mR7TSRM5+M37UfxO1YDSwUJbFwenuAnHxCLUjn1rq0foaNOs3AvgKEqMCSG2VC1Dre4z4HzBqNfBaJdQ+iKjVn9ow0LhLgtoX4CQMoJa4x/7BbRnV7Cta8ZOPWl4uJooHDnMUp0dtU4ezy/Ec8gS1yoWWUskAtWRP+2yVtmqTizaLrY1T+3F9kwIg+hepRaBLU6tMUio5bWvJ0Yyffygb7xXgObZo5TxkiNoLM5miq0Ju7n+TWqaSozlYaWoj+phW6xRGIrVZrhgTpBZ4C4CoJLW/uPmRUMgwNHlqlTOqkrWDSo5S64RKODCOJexrY3taSq1UdUj11P5iNErsRgGmhuAavZE0tSzqHjkWjlK7DE3hMqChhMa360S+lVziG+QhX8SK2gTUCj1kOF/1j25+CKYJlRyllp7OUFMb5qwVjiGnS+zlemBD1I6jW4TzLkat3L72N48HKg9ZMLAU3VDTRaml64vk9x+sY+b6qyTmqVICuZac4mgUy/hCRYXU/uKhXtWBxgB0soeZJ1FqaQTKngehKLrEiv4m0zLJrOxsEcaQRdRyjgeMJpqr9CePBwLcMpXcj4zsU0sXqxM/pynYpg2ow5YMg7C7zEOtcFbQN/2bWRYVH+od0GYeSOcwsk8tNbEkbBwJ+RZr9wQ0eiRtpGRAXXneI4uoFZzXIlRUcxRPHgbOviyXWlYm5RfdxtLe6LtXIge38kyA+ARaZcs63fQmeVkWLJUPuqIJRGmWxW+mvbVUiAeGUqn1qypYhVeMWpKlRvRltDK+wLIFm6BLpkYp1AHm5UaxlCDIWRKlVbFFBOvHCtPe4Bsql1rlSQ03DjFqXx0WQY41PchrbUdwq2zZwpCeIC2REQgqZFFGI31m5Hol1N5XmKyavBdW4dVWEtSSwmmyamNurZ0vqXG+gLuWyO6UWR4y5x3T02cw7vAkEG3T1zIFr1VCLdtvwtdKpjZQydNkOdeW2dp4q5JcWY2pcs6MptYPwOHV5GwHA173Y3eoKC/juwpqXzghktKp9QuIJ0lqBxb1kDdxNnKo5DXWokj6N0T6gsKdDw3lgImiTR78hjHwxSqopWnI3CrYMqntsOLVswS1nkYma3QWp8PJ2u5pvsR/RUZ2jAe8TomC2a8pLor6UX3OtrYSaru8QojSqQ0rvFoxaj1VTKJRyQpqN1tMKt3L4qAB5GcJVZvoC3vhLT2WKYQpc66DVgG1t9zAZ/nUsh2/fpeg9tUhMeRB0gPKUok5THegyaGPfTWGRh5YtR1CQYu3s/WjccibroBaNtEw018BtQ/MING5HzlvcsiZ22uKHGluBb91u5S/eVYVnEqcZmDZmdjpgX/YCx/b0SdGSzTLp7bN5hm3T0y51Ibt0OLUru11rIwj5FbKs51/oY0aKeSblCghfZBe/UDKXOKiYPXtPWv2g73L0qltsxmKNgWrgtpIf5oItXN3GSlhj6rSmXAPNJzxic3QyIKiy1oIpRcYa7SFLGiKCSL6wFpOaYi7Wjq1D0EPsikmXAm1DyZArfJGUsKhHpyWveeROx8seI1z2RDZOp5csRtMt2xjn+ucYyGWSJESHbP+nTq+pyqV2s6l31ES7GDFUAm1foVXgtq5O0wnvfjMOGvEC+K1zI0ga50J83iaunFzsFT9G7/Roskt0AREHz66/gNzToXK69H4Mr4J+n1qnN6q5SXQxHCvp6lV3tacWJK7WD/G+Rl97/iNrnMvWoUcv/m9v0y1d37Zumz3zPATQemtz20o+nRnaEE7c44oS62bdHGkzQDz2mIyk6ZhhG16TV7XXJbRyPm+LugNiqj1NV6c2hGJKuD7F8t2G5uv/eD9/X2w/9o0XNuR/emYHNk402aQxairqqaG/ZC1rlCFxUQ9yYMob4lAWZMxAMqc2+raUxy4NlYkWl3r4OZdRG2gkuPJdoPvZFZ4mt8sPyoRwMnz++KdQP/G3xbqBolFOWF8RYJaIPbJo1Y3J/ymDeIu5vmoDRoWxufVO66RiyBnAdHtXYIh1biW7IMAiPYEoqVSq6tGV9RtvTJqWbwuQS3xlcpnNn9vsdsn1dBUVgmiGc3zDA0uEqLg76rEcGZofJjpN935B/xDw7w7mwrv8EL0hRpYLXr/j2EY/+M26LkkIwP6LRVrLIrsPlQU01a7d3933zv/yNy4JBRtyYjetkQAUpwugT8b91+kygzFXwgGZjoU3JE7VDhtDkacQHAu1L9OfDJI9zMvhPqHME8Ho1KpdQu2FqtRJnYlOsl29t9uqlEhkB9zygGnSDJzjfIh+N1EeRTvGVejZLyXo5LzpM3VqBiCcKMcnHrNniLgAoBMsOvfET9JjJASAHm4ec4EavwCCnJryXU3qHEMIEVZcrA3ZTQnqlEVPnP7yXatjE8cnGIA/pKtw8Ynj29ePQACp1FHjf8E9nJ5bQdis/5yeI2jQVQWkFixg+Ldz2v8GlZvtpRattxtrYr/GkbvS9HStewGloVe47SxGmxdzOxajrvcl9GJtcaRMHrczyzXdhzrAMe2nc3+pzaw/wKMvn92+7fnz9ls9vm1HuyGddSpRo0aNUrD/wFfWy2dIyurzAAAAABJRU5ErkJggg==",
    "url": "https://golden.com/home",
    "authorization": [
        {
            "authtype": "API Key",
            "authattributes": [
                {
                    "key": "key",
                    "displayname": "Key"
                },
                {
                    "key": "value",
                    "displayname": "Value"
                }
            ]
        }
    ],
    "partner": "services",
    "account": "services",
    "isPromoted": false,
    "id": 500
}